### ISSUE TITLE
Feature: Check for session data json in relative entry directory

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1262]
This is so systems using systemctl to manage the service can reauthenticate easily

**Comment by smashah** [comment]
E.g
```
Project
 |
 +-- file 1
	
```

Before if your session data

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.15](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.14...3.1.15) (2021-01-20)

- Release 3.1.15 [`10afe92`](https://github.com/open-wa/wa-automate-nodejs/commit/10afe923ddcde741a5cd406233b5638af0136193)
- feat: library now checks for `.data.json` files in `require.main.path` also #1262 [`7b571f6`](https://github.com/open-wa/wa-automate-nodejs/commit/7b571f6d5175119d837e30adcd2f893a795c1d02)
- fix: #1258 patched [`f6d9b64`](https://github.com/open-wa/wa-automate-nodejs/commit/f6d9b64faf58db1bd60b47211a8f86f86d528595)
